### DISCUSSION TITLE
Is it possible to set a custom tag name for envPrefix?

### DISCUSSION THREAD

**Discussion by doxsch** [caarlos0/env#301] [Q&A] [answered]
We ask us if it is possible to define another tag e.g. `env-prefix` instead of `envPrefix`. This would make a migration to this lib much simpler. The same applies to the other tag names e.g. `envDefault`.

**Comment by caarlos0** [comment, marked as answer]
sorry, not possible.

**Reply by doxsch** [reply]
Hi @caarlos0 
Thank you for the fast reply. 

If we provide a PR, would that be a feature you would merge?

**Reply by caarlos0** [reply]
I'm willing to be convinced... I don't yet see how changing `envPrefix` to `env-prefix` improves things... 🤔

**Reply by doxsch** [reply]
Let me give it a try. 😉

We currently have a config struct that is not under our control. We have to fill it. However, so that we can use this library without copying the struct one-to-one and then assigning all the fields, it would be practical to adjust the tags and fill the struct directly. 
Also, this customization would align the features. If I have seen it correctly, you can currently overwrite the env tag e.g. with json.

**Reply by caarlos0** [reply]
isn't it easy enough to `s/env-prefix/envPrefix/g`?

**Reply by doxsch** [reply]
Unfortunately, as I said, the config struct is not part of the code that I have under my control. Otherwise, of course, it would be very easy as you suggested. 

But I have now decided to create my own config struct and tediously map it to the struct provided by the lib.
